### DISCUSSION TITLE
Daily 02/01/2024

### DISCUSSION THREAD

**Discussion by Sinan-Karakaya** [Sinan-Karakaya/R-Type#117] [Daily]
02/01/2024

**Comment by JasonLAGOUTE** [comment]
Hello ! Aujourd'hui on fais le ppt, on prépare la présentation et on speech cette aprem :)

**Comment by lucasdpt** [comment]
Diapo (imprévu) + Préparation de la présentation de cet aprem

**Comment by Sinan-Karakaya** [comment]
- Faire le PPT
- Régler quelques problèmes mineurs qui sont apparus
- Continuer sur mes tickets

**Comment by jovanHillion** [comment]
Hello, aujourd'hui je serai sur l'abstraction de la SFML, en comptant la présentation toute l'aprèm et (staffé ce matin)
